README.md

```diff
@@ -83,10 +83,39 @@ require 'qbwc'
 QBWC.add_job(:list_customers, false, '', CustomerTestWorker)
 ```
 
-After adding a job, it will remain active and will run every time QuickBooks Web Connector runs an update. If you don't want this to happen, you can have custom logic in your worker's `should_run?` or have your job disable or delete itself after completion. 
+After adding a job, it will remain active and will run every time QuickBooks Web Connector runs an update. If you don't want this to happen, you can have custom logic in your worker's `should_run?` or have your job disable or delete itself after completion. For example:
+
+```ruby
+	def handle_response(r, job, data)
+		QBWC.delete_job(job.name)
+	end
+
+```
+
 
 Use the [Onscreen Reference for Intuit Software Development Kits](https://developer-static.intuit.com/qbSDK-current/Common/newOSR/index.html) (use Format: qbXML) to see request and response formats to use in your jobs. Use underscored, lowercased versions of all tags (e.g. `customer_query_rq`, not `CustomerQueryRq`).
 
+### Requests passed to add_job ###
```

Comment:
I had to read this section a few times to understand what's going on. Can you revise it? Specifically:

- Why would one do this rather than using the method previously described? Consider summarizing this as the section title.
- The phrase "qbxml request" has not been used up to this point - use the language used by the rest of the document.
- Use inline code formatting when mentioning code, as the rest of the document does.